code/modules/research/techweb/all_nodes.dm

```diff
@@ -700,7 +700,7 @@
 	id = "mecha_tools"
 	display_name = "Exosuit Weapon (LBX AC 10 \"Scattershot\")"
 	description = "An advanced piece of mech weaponry"
-	prereq_ids = list("mecha", "adv_weaponry", "ballistic_weapons")

```

Comment:
scattershot is literally the most op weapon in terms of sustained damage.

Reply:
and so what, you want to put different prices on guns?

Reply:
ballistic weapons already has adv weaponry, its redunant

Reply:
knowing how techwebs work more than the author is concerning :^)